README.md

```diff
@@ -79,6 +80,44 @@ then add this to your maven configuration:
 </dependencies>
 ```
 
+### Java Gradle
+```groovy
+// Add plugin to classpath
+plugins {
+  id "net.ltgt.apt" version "0.5"
+}
+
+// Apply plugin
+apply plugin: 'net.ltgt.apt'
```

Comment:
You don't need that line with the `plugins` syntax.

FWIW, I like to link to the plugin portal for instructions: https://plugins.gradle.org/plugin/net.ltgt.apt